tests/test_coverage.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+from docstr_coverage import get_docstring_coverage
+
+SAMPLES_DIRECTORY = os.path.join("tests", "sample_files")
+EMPTY_FILE_PATH = os.path.join("tests", "sample_files", "empty_file.py")
+DOCUMENTED_FILE_PATH = os.path.join("tests", "sample_files", "documented_file.py")
+PARTLY_DOCUMENTED_FILE_PATH = os.path.join("tests", "sample_files", "partly_documented_file.py")
+SOME_CODE_NO_DOCS_FILE_PATH = os.path.join("tests", "sample_files", "some_code_no_docs.py")
+
+
+# @TODO: Should it check for module docs?
```

Comment:
@HunterMcGushion here's the potential bug I was referring to. Shouldn't it report a missing module doc in this case?

Reply:
To clarify, you're suggesting the log for the file should state both "File is empty" and "No module docstring"? I do understand that point, but in my opinion saying the file is empty already makes it clear that there's no module docstring. For the sake of being concise, [lines 233-241 in `coverage.py`](https://github.com/HunterMcGushion/docstr_coverage/blob/419eae9c5de7cfbdb38ff9c68705bdded1eb2f06/docstr_coverage/coverage.py#L233-L241) will log only one of {"File is empty", "No module docstring"}, or neither. Do you think logging both would be better?

Reply:
Sorry for not making it clear, and thanks for clarifying. The messaging is good as is.

I haven't realised that there is a special case of `File is empty`, which makes perfect sense. It's not reflected in the returned result dicts, that's what confused me.